<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KayitEkrani</class>
 <widget class="QWidget" name="KayitEkrani">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>2000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kayıt Ekranı</string>
  </property>
  <widget class="QPushButton" name="button_iptal">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>1620</y>
     <width>331</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>İptal</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>440</y>
     <width>901</width>
     <height>691</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">font: 75 25pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Ad:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit">
      <property name="styleSheet">
       <string notr="true">font: 75 18pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">font: 75 25pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Soyad:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="styleSheet">
       <string notr="true">font: 75 18pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="comboBox">
      <property name="styleSheet">
       <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <item>
       <property name="text">
        <string>Kız</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Erkek</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Diğer</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true">font: 75 25pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Doğum Tarihi:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDateEdit" name="dateEdit">
      <property name="styleSheet">
       <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_6">
      <property name="styleSheet">
       <string notr="true">font: 75 25pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Tel. No:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="styleSheet">
       <string notr="true">font: 75 18pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">font: 75 25pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>E posta:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="styleSheet">
       <string notr="true">font: 75 18pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_7">
      <property name="styleSheet">
       <string notr="true">font: 75 25pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Kullanıcı Adı:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="styleSheet">
       <string notr="true">font: 75 18pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_8">
      <property name="styleSheet">
       <string notr="true">font: 75 25pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Şifre:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="styleSheet">
       <string notr="true">font: 75 18pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_9">
      <property name="styleSheet">
       <string notr="true">font: 75 25pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Cinsiyet:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="button_kaydet">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>1620</y>
     <width>331</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
</string>
   </property>
   <property name="text">
    <string>Kaydet</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
